checkov/helm/runner.py

```diff
@@ -114,8 +114,8 @@ def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=R
                 f"Processing chart found at: {chart_dir}, name: {chart_meta['name']}, version: {chart_meta['version']}")
             with tempfile.TemporaryDirectory() as target_dir:
                 # dependency list is nicer to parse than dependency update.
-                proc = subprocess.Popen([self.helm_command, 'dependency', 'list', chart_dir], stdout=subprocess.PIPE, stderr=subprocess.PIPE)  # nosec
-                o, e = proc.communicate()
+                proc1 = subprocess.Popen([self.helm_command, 'dependency', 'list', chart_dir], stdout=subprocess.PIPE, stderr=subprocess.PIPE)  # nosec
```

Comment:
@metahertz why changing from poc to poc1?

Reply:
Good shout, not very readable! Thanks!